openedx/core/djangoapps/content_libraries/api/containers.py

```diff
@@ -437,6 +504,50 @@ def update_container_children(
                         changes=["units"],
                     ),
                 )
+        case ContainerType.Subsection:
+            units = []
+            for key in children_ids:
+                # Verify that all children are units
+                if (
+                    not isinstance(key, LibraryContainerLocator)
+                    or not key.container_type == ContainerType.Unit.value
+                ):
+                    raise ValueError(
+                        f"Invalid children type: {key}. All Subsection children must be Units",
+                    )
```

Comment:
Can you double-check if this is actually required? Based on https://github.com/openedx/openedx-learning/blob/8e38db5ddaba4e86f4840974439b4f7fbf6652d9/tests/openedx_learning/apps/authoring/subsections/test_api.py#L206-L239 I thought that Learning Core was already checking this.